Composer/packages/server/src/settings/index.ts

```diff
@@ -18,8 +19,12 @@ interface Settings {
   runtimeFrameworkVersion: string;
   botsFolder: string;
   appDataPath: string;
+  isWindows: boolean;
+  validDiskNames: string[];
 }
 
+const diskNames = ['C:/', 'D:/', 'E:/', 'F:/', 'G:/', 'H:/', 'I:/', 'J:/', 'K:/'];
```

Comment:
let's talk about this later, see if we can avoid a hard code here

Reply:
The suggestion is, no matter what the implementation is, it's would be necessary to update the protocol, adding new state or settings, we should use existing abstraction, and treat disk part in windows just as normal folders.